Cubical/Core/Prelude.agda

```diff
@@ -132,3 +137,9 @@ isProp A = (x y : A) → x ≡ y
 
 isSet : Set ℓ → Set ℓ
 isSet A = (x y : A) → isProp (x ≡ y)
+
+-- Equivalence on hSets are hProps
+
+isProp[Set≡] : ∀ {A : Set ℓ} → isSet A →
```

Comment:
This is just changing the implicits in the definition of `isSet`, so I don't understand why it should be necessary. Also it should not be added to the Core. Lemmas of this kind should go into `Foundations.HLevels`. However I really don't see why this should be necessary at all.

Reply:
Removed.